core/configuration/dt_config_provider.go

```diff
@@ -170,5 +171,6 @@ func validateConfiguration(config *DtConfiguration) error {
 }
 
 func generateAgentId() int64 {
-	return int64(rand.Uint64())
+	rng := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Is there no better standard source for a random seed, e.g. based on /dev/(u)random?
Depending on the actual timer resolution, UnixNano might return the same value for multiple lambdas that are started at about the same time (e.g. provisioned concurrency, or scale-up that launches multiple lambda instances "at once")